etc/ReplayOfflineConfiguration.py

```diff
@@ -107,9 +108,10 @@
 
 # Configure ScramArch
 setDefaultScramArch(tier0Config, "el8_amd64_gcc10")
+setScramArch(tier0Config, defaultCMSSWVersion['default'], "slc7_amd64_gcc10")
```

Comment:
@tvami are you sure the changes here are correct for Run 2 data?
It looks different from what we used in the past: https://github.com/dmwm/T0/pull/4674

Reply:
I'll fix it thanks!